java/src/main/java/com/logicalclocks/hsfs/metadata/TagsApi.java

```diff
@@ -87,9 +100,54 @@ public void add(TrainingDataset trainingDataset, String name, Object value)
         trainingDataset.getId(), name, value);
   }
 
+  public void add(FeatureView featureView, String name, Object value)
+      throws FeatureStoreException, IOException {
+    add(getFvUriTemplate(featureView, name), value);
+  }
+
+  private UriTemplate getFvUriTemplate(FeatureView featureView) {
+    return UriTemplate.fromTemplate(FV_TAGS_PATH)
+        .set("projectId", featureView.getFeatureStore().getProjectId())
+        .set("fsId", featureView.getFeatureStore().getId())
+        .set("fvName", featureView.getName())
+        .set("fvVersion", featureView.getVersion());
+  }
+
+  private UriTemplate getFvUriTemplate(FeatureView featureView, String tagName) {
+    return UriTemplate.fromTemplate(FV_TAGS_PATH)
+        .set("projectId", featureView.getFeatureStore().getProjectId())
+        .set("fsId", featureView.getFeatureStore().getId())
+        .set("fvName", featureView.getName())
+        .set("fvVersion", featureView.getVersion())
+        .set("name", tagName);
+  }
+
+  public void add(FeatureView featureView, Integer trainingDatasetVersion, String name, Object value)
```

Comment:
checkstyle complains: TagsApi.java:125: Overload methods should not be split. Previous overloaded method located at line '103'. [OverloadMethodsDeclarationOrder]